### PR TITLE
infoschema/perfschema: add cluster statement summary table

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 

Tests 

 - Unit test

Code changes

Release note

 - Support read cluster statement sumary table.

### CODE REVIEW

**Comment by shenli** [comment]
I can not understand the description of this PR.

**Comment by djshow832** [comment]
s/perschema/perfschema

File: infoschema/perfschema/const.go [hunk 1]
```diff
@@ -449,17 +451,27 @@ const fieldsInEventsStatementsSummary = " (" +
...
 // tableEventsStatementsSummaryByDigestHistory contains the column name definitions for table
 // events_statements_summary_by_digest_history.
 const tableEventsStatementsSummaryByDigestHistory = "CREATE TABLE if not exists " + tableNameEventsStatementsSummaryByDigestHistory +
-	fieldsInEventsStatementsSummary
+	fieldsInEventsStatementsSummary + ")"
+
+// tableClusterEventsStatementsSummaryByDigest contains the column name definitions for table
+// cluster_events_statements_summary_by_digest, same as MySQL.
+const tableClusterEventsStatementsSummaryByDigest = "CREATE TABLE if not exists " + tableNameClusterEventsStatementsSummaryByDigest +
+	fieldsInEventsStatementsSummary + ", ADDRESS VARCHAR(64) DEFAULT NULL)"
```
Comment (djshow832) [inline infoschema/perfschema/const.go]: I prefer to move this field to the front.

**Comment by coocood** [comment]
Since the name doesn't match an existing table in MySQL, can we make the table name shorter?

**Comment by crazycs520** [comment]
@coocood 

As you can see, The cluster table name only has a `cluster_` prefix compare to the memory table name, It easy for user to understand the usage of cluster table is same with the memory table. Use another short table name maybe confuse the user?

| memory table name                           | cluster table name                                  |
| ------------------------------------------- | --------------------------------------------------- |
| slow_query                                  | cluster_slow_query                                  |
| processlist                                 | cluster_processlist                                 |
| events_statements_summary_by_digest         | cluster_events_statements_summary_by_digest         |
| events_statements_summary_by_digest_history | cluster_events_statements_summary_by_digest_history |

File: infoschema/tables_test.go [hunk 2]
```diff
@@ -1032,13 +1032,28 @@ func (s *testClusterTableSuite) TestSelectClusterTable(c *C) {
 	prepareSlowLogfile(c, slowLogFileName)
 	defer os.Remove(slowLogFileName)
 	tk.MustExec("use information_schema")
+	tk.MustExec("set @@global.tidb_enable_stmt_summary=1")
```
Comment (djshow832) [inline infoschema/tables_test.go]: defer tk.MustExec("set @@global.tidb_enable_stmt_summary=0")

Comment (crazycs520) [reply infoschema/tables_test.go]: Have test after `set @@global.tidb_enable_stmt_summary=0`.

**Comment by coocood** [comment]
OK, we can add a view with a short table name later.

File: infoschema/perfschema/const.go (hunk 1 above)
Comment (crazycs520) [reply infoschema/perfschema/const.go]: @lonng What do you think？
If change this, need to change another cluster table too, Should better consistent with another cluster table,

Comment (lonng) [reply infoschema/perfschema/const.go]: Prefer to make the `address` as the first column.

Comment (crazycs520) [reply infoschema/perfschema/const.go]: Addressed, Both cluster table was changed. PTAL @djshow832

**Pull request merged**